src/citrine/builders/predictors.py

```diff
@@ -134,3 +144,112 @@ def _build_mean_property_predictor(
+        identifiers. de-duplication is performed if a uid or identifier is shared between two or
+        more rows. The content of a de-duplicated row will contain the union of data across all
+        rows that share the same uid or at least 1 identifier. Training data is unnecessary if the
+        predictor is part of a graph that includes all training data required by this predictor.
+
+    Returns
+    -------
+    GraphPredictor
+        GraphPredictor connecting all inputs to all latent variables and all inputs and latent
+        variables to all outputs.
+    """
+    ml_model_feature_descriptors = [d for d in inputs
+                                    if isinstance(d, (RealDescriptor, CategoricalDescriptor))]
+    chemical_formula_descriptors = [d for d in inputs
+                                    if isinstance(d, (ChemicalFormulaDescriptor))]
+    molecular_structure_descriptors = [d for d in inputs
+                                       if isinstance(d, (MolecularStructureDescriptor))]
+
+    chemical_formula_featurizers = [
+        ChemicalFormulaFeaturizer(
+            name=f'{chemical_formula_descriptor.key} featurizer',
+            description='',
+            input_descriptor=chemical_formula_descriptor,
+            features=['standard'],
+        ) for chemical_formula_descriptor in chemical_formula_descriptors
+    ]
+
+    molecular_structure_featurizers = [
+        MolecularStructureFeaturizer(
+            name=f'{molecular_structure_descriptor.key} featurizer',
+            description='',
+            input_descriptor=molecular_structure_descriptor,
+            features=['standard'],
+        ) for molecular_structure_descriptor in molecular_structure_descriptors
+    ]
+
+    # TODO: Fix needing to pass the project (if possible)
+    featurizer_responses = []
+    for featurizer in chemical_formula_featurizers + molecular_structure_featurizers:
+        featurizer_responses.extend(
+            project.descriptors.from_predictor_responses(
+                predictor=featurizer,
+                inputs=[featurizer.input_descriptor],
+            )
+        )
+
+    automl_lv_predictors = [
+        AutoMLPredictor(
+            name=f'{latent_variable_descriptor.key} Predictor',
+            description='',
+            inputs=ml_model_feature_descriptors + featurizer_responses,
+            output=latent_variable_descriptor,
+        ) for latent_variable_descriptor in latent_variables
+    ]
+
+    automl_output_predictors = [
+        AutoMLPredictor(
+            name=f'{output_descriptor.key} Predictor',
+            description='',
+            inputs=ml_model_feature_descriptors + latent_variables + featurizer_responses,
+            output=output_descriptor,
+        ) for output_descriptor in outputs
+    ]
+
+    predictor = GraphPredictor(
+        name=name,
+        description=description,
+        predictors=[
```

Comment:
Nitpick: Same as with the input lists above, you can concatenate these lists instead of unpacking and repacking them.